src/wwwroot/services/settings.js

```diff
@@ -137,5 +138,18 @@
 
       return plansCache;
     }
+
+    function getCountries() {
```

Comment:
yes, because injecting `sign up` service in the `billing `controller is not logic, I prefer that the code writing is clear, I will work about this